README.rst

```diff
@@ -265,3 +265,7 @@ application (and the "blank" white screen is a torch application):
 .. image:: res/BeaconApp.png
    :alt: Flash the relatively powerful HRS LED repeatedly
    :width: 179
+
+.. image:: res/FlappyApp.png
```

Comment:
It has to be consistent with everything else, so `res/FlappybirdApp.png`, also rename the file